src/components/donation/Steps/common/Summary.tsx

```diff
@@ -51,29 +54,43 @@ export default function Summary({
             : "Total donation"}
         </dt>
         <Amount amount={props.amount} classes="text-navy-d4" />
-        <div className="flex items-center justify-between col-span-full">
-          <dt className="mr-auto text-sm">Sustainability Fund</dt>
-          <Amount classes="text-sm" amount={locked} />
-        </div>
-        <div className="flex items-center justify-between col-span-full">
-          <dt className="mr-auto text-sm">Direct Donation</dt>
-          <Amount classes="text-sm" amount={liq} />
-        </div>
-        {props.tip && (
+        {locked > 0 && (
```

Comment:
comment: 
for me, summary is a step that lays out prior choices I've made 
e.g. opting for `100%` liquid  and `0%` locked and it's justifiable that this choice is presented to me again (summary)

<img width="792" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/5f08164f-6a01-48a2-ad99-3fb9bec17c0e">

compared to (the intent of donating is lost) and it's just assumed that user remembers that this donation is all-in liquid
<img width="822" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/0b553fd6-695c-4261-b713-17484f5fc108">

Reply:
This is a fair point.